modules/wezterm/hm.nix

```diff
@@ -68,87 +68,109 @@
-          let
-            inherit (config.stylix) fonts;
-          in
-          lib.mkForce ''
-            -- Generated by Stylix
-            local wezterm = require("wezterm")
-            wezterm.add_to_config_reload_watch_list(wezterm.config_dir)
-            -- Allow working with both the current release and the nightly
-            local config = {}
-            if wezterm.config_builder then
-              config = wezterm.config_builder()
-            end
-            local stylix_base_config = {
-                color_scheme = "stylix",
-                font = wezterm.font_with_fallback {
-                    "${fonts.monospace.name}",
-                    "${fonts.emoji.name}",
-                },
-                font_size = ${builtins.toString fonts.sizes.terminal},
-                window_background_opacity = ${builtins.toString config.stylix.opacity.terminal},
-                window_frame = {
-                    active_titlebar_bg = "${base03}",
-                    active_titlebar_fg = "${base05}",
-                    active_titlebar_border_bottom = "${base03}",
-                    border_left_color = "${base01}",
-                    border_right_color = "${base01}",
-                    border_bottom_color = "${base01}",
-                    border_top_color = "${base01}",
-                    button_bg = "${base01}",
-                    button_fg = "${base05}",
-                    button_hover_bg = "${base05}",
-                    button_hover_fg = "${base03}",
-                    inactive_titlebar_bg = "${base01}",
-                    inactive_titlebar_fg = "${base05}",
-                    inactive_titlebar_border_bottom = "${base03}",
-                },
-                colors = {
-                  tab_bar = {
-                    background = "${base01}",
-                    inactive_tab_edge = "${base01}",
-                    active_tab = {
-                      bg_color = "${base00}",
-                      fg_color = "${base05}",
-                    },
-                    inactive_tab = {
-                      bg_color = "${base03}",
-                      fg_color = "${base05}",
-                    },
-                    inactive_tab_hover = {
-                      bg_color = "${base05}",
-                      fg_color = "${base00}",
-                    },
-                    new_tab = {
-                      bg_color = "${base03}",
-                      fg_color = "${base05}",
-                    },
-                    new_tab_hover = {
-                      bg_color = "${base05}",
-                      fg_color = "${base00}",
-                    },
+      }
+    )
+  ];
+  generalConfig =
+    {
+      fonts,
+      colors,
+      opacity,
+    }:
+    with colors;
+    let
+      indent =
+        string: "    " + lib.concatStringsSep "\n    " (lib.splitString "\n" string);
+    in
+    lib.mkIf (fonts != null || colors != null || opacity != null) {
```

Comment:
Looks like there's a change in behaviour here: previously this file would only be created if `config.programs.wezterm.enable` is `true`.

This is an optimization to avoid creating unnecessary files for people who don't use WezTerm.